### PR TITLE
[fr] Unify links to Mozilla Firefox source code

### PR DESCRIPTION
**Pull request by leon-win**
### Description

This PR unifies links to Mozilla Firefox source code by using searchfox.org instead of hg.mozilla.org and obsolete dxr.mozilla.org, mxr.mozilla.org for `fr` locale.

### Related issues and pull requests

Relates to https://github.com/mdn/content/pull/33585

### CODE REVIEW

File: files/fr/mdn/writing_guidelines/writing_style_guide/code_style_guide/shell/index.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Les directives suivantes expliquent comment rédiger des exemples de lignes de c
 Un _shell_ est un programme qui attend que vous tapiez une commande et que vous appuyiez sur la touche retour. Pour indiquer les commandes que vous devez taper, la documentation MDN les répertorie dans un bloc de code, similaire aux exemples de code. Un tel bloc ressemble à ceci :
 
 ```bash example-good
-# Cela peut prendre un certain temps…
-hg clone https://hg.mozilla.org/mozilla-central/ firefox
-cd firefox
+# This may take a while...
```
Comment (cw118) [inline files/fr/mdn/writing_guidelines/writing_style_guide/code_style_guide/shell/index.md:15]:
```suggestion
# Cela peut prendre un certain temps…
```

**Review by cw118 — CHANGES_REQUESTED** [review]
Thank you @leon-win ! Just one small comment :)

**Comment by leon-win** [comment]
> Thank you @leon-win ! Just one small comment :)

Done :ok_hand: ))

**Review by cw118 — APPROVED** [review]
This is great, thank you so much!

**Pull request merged**